packages/nestjs/src/decorators.ts

```diff
@@ -77,7 +77,7 @@ export function SentryExceptionCaptured() {
         return originalCatch.apply(this, [exception, host, ...args]);
       }
 
-      captureException(exception);
+      captureException(exception, { mechanism: { handled: false, type: 'auto.function.nestjs.exception_captured' } });
```

Comment:
I think `auto.function.*` is correct here, given this decorator can be applied to any arbitrary function.